VIDEO_5_SCRIPT.md

```diff
@@ -0,0 +1,71 @@
+
+With these ideas and principles clear, after a long time of searches and negotiations, AsyncAPI joined the Linux Foundation umbrella in March 2021. This move granted neutrality and open governance, ensuring that the specification will continue to be driven democratically by the people who contribute their time and effort to the project.
+
+## What is transparency
+
+Organizational transparency can be defined as a deliberate attempt by the organization to create an environment of trust in which to promote free access to information, open communication, and the participation of all professionals in decision-making. 
+
+When we talk about transparency, we refer to that attitude and methodology that allows controlling the financial aspects of a company to fulfill its mission and vision with the participation of all the people in an organization. Transparency requires obligations, clarity, and implication to sustain it. The information provided must be truthful, based on evidence that can be checked and verified.
+
+Transparency has become a fundamental concept in the democratization processes undertaken by governments, institutions, companies, organizations, projects... in recent years. In its ideal state, democracy is nothing more than a method that allows the community members to analyze reality from different points of view to make decisions freely and for the common good.
+
+## What is open governance
+
+Open governance is a model of governance that promotes the principles of transparency, integrity, accountability, and stakeholder participation in support of democracy and inclusive growth. Driven by the participation of all the people that make up a project, company, or institution, it promotes engagement and democracy.
+
+In the same way, this model facilitates working with like-minded people living thousands of miles away to pursue and achieve a common goal. The intention is to break down individualism and hierarchical archetypes while upholding a system that gives autonomy and decision-making power to its members.
+
+Therefore, all relevant project data are public. The developed tasks are accessible and open to the participation of the community. This is intended to cultivate an environment of trust and respect for the user that encourages involvement and collaboration. In this atmosphere, adaptable and flexible, nothing is imposed or taken for granted.
+
+## Why AsyncAPI is committed to an Open Governance model
+
+Partnering with Postman was a crucial moment for assuring the efforts of the AsyncAPI Initiative. This event was the propellant to accelerate the transfer of AsyncAPI to an open government foundation to assure the community that a single company does not control the AsyncAPI Initiative. In doing so, the specification safeguarded against private interests.
```

Comment:
Partnering with Postman ensures that the AsyncAPI initiative is ruled under an open governance model and that the control over the initiative is distributed across companies.

Reply:
It is not the partnership with Postman that ensures that an open government model drives AsyncAPI. It is the reason for seeking a foundation to host the initiative, thus ensuring that the project will be controlled by its community and not by companies.